ckanext/dcat/profiles.py

```diff
@@ -586,6 +619,25 @@ def parse_dataset(self, dataset_dict, dataset_ref):
         '''
         return dataset_dict
 
+    def _extract_catalog_dict(self, catalog_ref):
```

Comment:
Considering that this data gets stored in every single dataset I think we should limit by default the fields that get stored. I understand that this data is used when serializing the catalog, but it seems really inefficient to store all these extras. I'd suggest storing only the remote catalog URI and maybe title to help with faceted search. Description, language and modified seem like secondary information, and the publisher JSON blob definitely seems like pushing it. Users can implement a custom profile if they want this extra fields or implement some lookup table.

Reply:
According to DCAT specifications, dct:description and dct;publisher are mandatory properties for the Catalog.

I agree for the customization of the other fields.